### DISCUSSION TITLE
Any plans adding DMM game player to this?

### DISCUSSION THREAD

**Discussion by TheFeri** [moraroy/NonSteamLaunchers-On-Steam-Deck#70] [Ideas]
That thing is a pain in the ass to install more so than anything else. Once I hardly managed to make it work with lutris but it broken when lutris integrated proton and can't get it to work again or even install it again. Would be nice if this would include that too.

**Comment by moraroy** [comment]
@TheFeri what is dmm player?

**Reply by TheFeri** [reply]
https://player.games.dmm.com/

This thing. No wonder you don't know about, there's pretty much no english games there by default.

**Reply by moraroy** [reply]
@TheFeri lol is that an android emulator?

**Reply by TheFeri** [reply]
no this is a store just like steam. It has PC games too. Infact it has a ton visual novels that only have fan translations and is the only offical way to get them.

**Reply by moraroy** [reply]
@TheFeri I just installed it perfectly fine on my deck thing is I don't have an account and don't know Japanese lol

**Reply by TheFeri** [reply]
huh weird I just get an error all the time... eh... ah whatever. Well thanks for your time I guess.

**Reply by moraroy** [reply]
@TheFeri I can add it...but I just cant test anything further I'm trying to sign up for an account but I can't seem to fo some reason lol

**Reply by TheFeri** [reply]
Well I'm pretty sure if it starts up it'll work just fine after that. It took me a while to make an account back in the day too. Don't worry about it.

**Reply by moraroy** [reply]
@TheFeri ah ok..OK... just to let you know, I installed it using GE-Proton8-4, and had no problems installing.

**Reply by TheFeri** [reply]
as a non steam game right? I don't really like that method I prefer lutris but get an error with that with the same proton... Linux can be weird sometimes I guess.

**Reply by moraroy** [reply]
@TheFeri yeah I installed it as a non Steam game, I don't like lutris lol... you know this NonSteamLauncher Script installs all these launchers as nonsteam games right?

**Reply by TheFeri** [reply]
Yeah, it does is automatically so I don't mind but I hate doing that manually. I just don't have the patience to do it like that. I personally hate bottles because that thing literally never worked when I tried to use it.

**Reply by moraroy** [reply]
@TheFeri haha yeah bottles is kinda weird lol I once used that thing to try and instal the Xbox Game app in it.. I got really close but ran out of space on my 64gb Steam deck

**Reply by TheFeri** [reply]
Yeah bottles is weird, but I only had 2 problems with lutris. And as I said even this worked for months then lutris rolled a big update to allow the usage of proton and it just stopped working. I prefer lutris because most of the time I need more fiddleing with prefixes than what installing as a non steam game lets me or just need to run more than 1 things at once, in the same prefix.

**Reply by moraroy** [reply]
@TheFeri I see yeah that's true... I'm a noob when it comes to understanding prefixes and wines but so far I just hope people are able to play games whether it's through this script or lutris or heroic

**Reply by TheFeri** [reply]
Well I still live off from guides and help from other people but you did this whole thing so I think you are doing better than me lol.

And if I uderstood correctly, in the end you will add DMM in right?

**Reply by moraroy** [reply]
@TheFeri once it's added in what games are you able to play on there?

**Reply by TheFeri** [reply]
I mainly want Alice gear aegis.  And well since it's a japanse store who knows what untranslated VN will catch my eye in the future.

**Reply by moraroy** [reply]
@TheFeri I'll add it in the next update but you're the only one that can test how things work on there since I have no idea what's it saying haha

**Reply by TheFeri** [reply]
Alright just tell me when's that and I'll check as soon as I can and give feedback

**Comment by moraroy** [comment]
@TheFeri I added it!!

**Reply by TheFeri** [reply]
Alright, IF I don't have to work on saturday I'm gonna check it and tell you if its working right or not.

**Reply by TheFeri** [reply]
Alright, so it seems it installs perfectly, I can login perfectly and can install the game perfectly but it just does't open(the game) after that...
I wanted to try and install something with protontricks but protontricks doesn't even sees it so I'm out of ideas.

**Reply by moraroy** [reply]
@TheFeri dmm game launcher seems to be region locked now, I can't get the launcher url to download it, do you know another source or link that would work in the u.s.?

**Reply by TheFeri** [reply]
Were you doing it on linux? I don't think its region locked because on my steam deck I can't download it either but I can on my old windows laptop. It simply detects from the browser you aren't running windows. Saw some other sites doing this too.

**Reply by OcRazor** [reply]
May try this link
https://apidgp-gameplayer.games.dmm.com/archive/latest?app=dgp5win

or this (may need login)
https://accounts.dmm.com/service/login/password/=/path=https%3A%2F%2Fsite-gameplayer.dmm.com%2Fgameplayer%2Fgame%2Finstaller%2Fwin%2Fx86%2Falicegearaegisexe%2Fcl%2Fgeneral

I can download the launcher by those 2 link in SD browser (I'm use Brave)

If can't download may need use VPN to load it.

But I'm tested your app in older version, it can download and install DMM game player in normal.
seem no problem with me in download issue.

And I'm also tested the game Alice Gear Aegis.
It can run but in black screen, if it start from DMM launcher.
But I'm try the app run up from steam compatibility layer. (in my situation I'm use GE-Proton 8-9)
Still black screen, but it have the anime pointer.
(seems some difference between that, I'm not coder dunno how to solve it)

May those information can help.
If need any test on steam deck, I can help with that.

Thank you guys.

**Reply by OcRazor** [reply]
Well.
Last night I tried delete all folder (pfx/drive_c) and reinstall the DMM launcher from NonSteamLaunchers.
Now the issue shows up.
I can't download and install either.

But I saw you add the URL in your code. (line 1528)
I don't know what difference between newer version and old version...

Just report my test situation, please be free in your APP development.

**Reply by moraroy** [reply]
@OcRazor i cant seem to find DMM Games Launcher online, like a good url to download it from i get acces denied, i think its behind an account creation right?

**Comment by deckboy6253** [comment]
Can you guys add magic the gathering online.
